src/couch_replicator_tasks.erl

```diff
@@ -0,0 +1,85 @@
+-module(couch_replicator_tasks).
```

Comment:
Agree with @rnewson this could be a function. To avoid having to export job record and having an extra supervised gen_server, we could make a to_ejson() function in scheduler.  Then it can look at the ets table and read it (it is protected by default so reads would work).

We'd have to think about if that is called during a time when ets table is not there (due to applications started after chttpd, or during restart) and perhaps handle it if we think it is meaningful (return empty list or some error)